src/aggregator/integration/one_client_multi_type_untimed_test.go

```diff
@@ -1,4 +1,4 @@
-// +build integration
```

Comment:
Why doesn't this need `// +build integration` whereas both are included in other files?

Reply:
Doh, this is more "stuff my IDE did for me". I'm standardizing it now (I guess `//go:build` is the new norm, so moving to that).